src/bootstrap/doc.rs

```diff
@@ -82,6 +82,7 @@ book!(
     Reference, "src/doc/reference", "reference", submodule;
     RustByExample, "src/doc/rust-by-example", "rust-by-example", submodule;
     RustdocBook, "src/doc/rustdoc", "rustdoc";
+    StyleGuide, "src/doc/style-guide", "style-guide";
```

Comment:
If you want to avoid this being distributed through rustup, you can make the `DEFAULT = true` constant configurable in the macro, and disable it for the style guide: https://github.com/rust-lang/rust/blob/39484ac3b0b80c1c98c4d46a9f88336c02f24da6/src/bootstrap/doc.rs#L42
But I think distributing it is fine.

Reply:
@jyn514 Happy to have it distributed, if you don't think that'll be an issue.